lib/Doctrine/ORM/Mapping/AttributeOverride.php

```diff
@@ -19,6 +19,8 @@
 
 namespace Doctrine\ORM\Mapping;
 
+use Attribute;
```

Comment:
@carusogabriel do you have a plan or a workaround for me how to get this working for now? How would we handle this in PHPCS?

Reply:
No, sorry, I haven't been looking at that. After PHP 8.0's release, I might have some time. For now, I'd just put this file on `phpcs.xml` ignoring whatever rule is causing it, with a comment "waiting for PHP 8.0 compatibility" :)